pallets/gear-scheduler/src/lib.rs

```diff
@@ -144,15 +136,15 @@ pub mod pallet {
 
     // ----
 
-    // Private storage for missed blocks collection.
+    // Private storage for the first missing block.
     #[pallet::storage]
-    type MissedBlocks<T> = StorageValue<_, MissedBlocksCollection<T>>;
+    type FirstMissingBlock<T> = StorageValue<_, BlockNumberFor<T>>;
```

Comment:
Rename as well and note that this PR will require database migrations.
Let's train it within the task!